src/components/Player.js

```diff
@@ -144,7 +144,6 @@ export default class Player extends Component {
       <Bezel key="bezel" order={3.0} />,
       <BigPlayButton key="big-play-button" order={4.0} />,
       <ControlBar key="control-bar" order={5.0} />,
-      <CueBar key="cue-bar" order={6.0} />,
```

Comment:
removing this since we are importing `CueBar` in the app and leaving this here causes duplicates.